contracts/ckb-combine-lock/src/child_script_args.rs

```diff
@@ -0,0 +1,414 @@
+use alloc::{string::String, vec::Vec};
+use ckb_std::ckb_types::{bytes::Bytes, core::ScriptHashType, packed::Byte32, prelude::*};
+
+pub struct ChildScriptArgs {
+    code_hash: Byte32,
```

Comment:
Use [u8; 32]. We then need another function to convert `ChildScript` to `ChildScriptEntry`.